### DISCUSSION TITLE
What/ Where can I find Pose Style examples?

### DISCUSSION THREAD

**Discussion by Acephalia** [OpenTalker/SadTalker#466] [Q&A]
Hello, I've been trying to figure out what the pose styes are and have pretty much looked all over. I found [this thread](https://github.com/OpenTalker/SadTalker/issues/120) that says the pose styles are in the project page but I cannot for the life of me find this. Some help to a link to where I can see the pose styles or an explanation as to what they do would be absolutely fabulous. Thank you.

**Comment by jjmlovesgit** [comment]
Here are some example files:  https://github.com/OpenTalker/SadTalker/tree/main/examples/ref_video

....also some context for them and how to use them -- you want to use the newer interface offs the dev branch as explained here:

https://github.com/OpenTalker/SadTalker/discussions/386

**Comment by G-force78** [comment]
Does anyone have a good source for some blinking templates?

**Comment by G-force78** [comment]
You can find some pose videos to use here
https://lalamu.studio/demo
https://github.com/OpenTalker/DPE
https://replicate.com/yoyo-nb/thin-plate-spline-motion-model
https://replicate.com/wyhsirius/lia/examples#dm6lpp75yne47m4q4natr472m4

**Comment by afgonczol** [comment]
Did you ever find an answer?

**Reply by Acephalia** [reply]
Never did unfortunately.
